2021/OffSeason2021/src/main/cpp/subsystems/Drivetrain.cpp

```diff
@@ -603,7 +606,7 @@ void Drivetrain::MoveWithLimelightInit()
 void Drivetrain::MoveWithLimelightExecute(double tx, double ty, bool tv)
 {
     // get turn value - just horizontal offset from target
-    double turnOutput = -m_turnController.Calculate(tx);
+    double turnOutput = -m_turnController.Calculate(tx, m_targetAngle);

```

Comment:
Does the present implementation allow us to point the robot at the goal, measure the angle, and put that angle directly into the dashboard (and ultimately the config file)? The sign should match what we read during calibration (or will need to be inverted somewhere in the code). This is what Adam was asking tonight on the field.